config/nais.yml

```
@@ -102,6 +102,14 @@ spec:
     - name: {{ item.name }}
       value: "{{ item.value }}"
     {{/each}}{{/if}}
+{{#if maskinporten}}
+  maskinporten:
+    enabled: true
+    scopes:
+      consumes: {{#each maskinporten.consumes as |consume| }}
+         - name: "{{ consume }}"
+   {{/each }}
+{{/if }}
```

```suggestion
  {{#if maskinporten}}
  maskinporten:
    enabled: true
    scopes:
      {{#each maskinporten.consumes as |consume| }}
      consumes:
         - name: "{{ consume }}"
      {{/each }}
  {{/if }}
```
Så er formatet likt resten.